crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -440,6 +442,21 @@ fn repeat_layout(layout: &Layout, n: usize) -> Option<(Layout, usize)> {
     }
 }
 
+// TODO: replace with `Layout::pad_to_align` if/when it stabilizes
```

Comment:
Is there a tracking issue for this?

Reply:
The utility methods for `Layout` are part of [alloc_layout_extra](https://github.com/rust-lang/rust/issues/55724)

Reply:
```suggestion
// TODO: replace with `Layout::pad_to_align` if/when it stabilizes (https://github.com/rust-lang/rust/issues/55724)
```